packages/editor-common/web/src/Utils/tiptap/types.ts

```diff
@@ -1,8 +1,7 @@
-import { RicosTiptapExtension, TiptapExtensionConfig } from 'wix-rich-content-common';
+import { RicosTiptapExtension, RicosExtension } from 'wix-rich-content-common';
 
-export interface CreateExtensionParams<T extends TiptapExtensionConfig>
-  extends Omit<RicosTiptapExtension<T>, 'type'> {}
+export type CreateExtensionParams<T extends RicosExtension> = Omit<RicosTiptapExtension<T>, 'type'>;
 
-export type CreateExtension<T extends TiptapExtensionConfig> = (
+export type CreateExtension<T extends RicosExtension> = (
   params: CreateExtensionParams<T>
 ) => RicosTiptapExtension<T>;
```

Comment:
`RicosTiptapExtension` is unnecessary

```suggestion
export type CreateExtension<T extends RicosExtension> = (
  params: CreateExtensionParams<T>
) => T;
```